chainercv/experimental/links/model/pspnet/pspnet.py

```diff
@@ -305,8 +305,13 @@ class PSPNetResNet101(PSPNet):
     _models = {
         'cityscapes': {
             'param': {'n_class': 19, 'input_size': (713, 713)},
-            'url': 'https://github.com/yuyu2172/share-weights/releases/'
-            'download/0.0.6/pspnet_resnet101_cityscapes_convert_2018_05_22.npz'
+            'url': 'https://chainercv-models.preferred.jp/'
+            'pspnet_resnet101_cityscapes_trained_2018_12_19.npz',
+        },
+        'ade20k': {
+            'param': {'n_class': 150, 'input_size': (473, 473)},
+            'url': 'https://chainercv-models.preferred.jp/'
+            'pspnet_resnet101_ade20k_trained_2018_12_23.npz',
```

Comment:
How about updating this block? https://github.com/chainer/chainercv/blob/f43f891bada81dbf2e5d333e6732e574f97579c2/chainercv/experimental/links/model/pspnet/pspnet.py#L131-L138